### PR TITLE
Fix placeholder re-name functionality

### PR DESCRIPTION
**Pull request by alex-deaconu**
[stage-2]

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc this is the initial PR with unit/e2e test coverage.

From here as discussed this goes over to @ghislaineguerin for UX.

Please review what I have so far. Thanks.

File: partials/placeholder-settings.html [hunk 1]
```diff
@@ -1,17 +1,45 @@
...
+      </a>
+
+      <div class="input-group">
+        <input type="text" id="placeholderNewId" name="newId" class="form-control" placeholder="Placeholder Name" ng-model="placeholder.newId" required placeholder-name-validator>
       </div>
+      <p id="requiredValidation" class="text-danger" ng-show="!placeholderFields.newId.$pristine && placeholderFields.newId.$error.required">
+        Required
+      </p>
+      <p id="invalidValidation" class="text-danger" ng-show="!placeholderFields.newId.$pristine && placeholderFields.newId.$error.alphanumeric">
+        Placeholder name contains invalid character
```
Comment (ezequielc) [inline partials/placeholder-settings.html]: @alex-deaconu I think after @ghislaineguerin finishes, we need to extract these strings to i18n. Cheers

File: test/e2e/test-suite.scenarios.js [hunk 2]
```diff
@@ -6,19 +6,21 @@ ArtboardScenarios = require('./cases/artboard.js');
...
-artboardScenarios = new ArtboardScenarios();
-presentationPropertiesScenarios = new PresentationPropertiesScenarios();
-artboardPlaceholdersScenarios = new ArtboardPlaceholdersScenarios();
-placeholdersListScenarios = new PlaceholdersListScenarios();
-playlistScenarios = new PlaylistScenarios();
-templateAddScenarios = new TemplateAddScenarios();
-htmlEditorScenarios = new HtmlEditorScenarios();
-presentationAddScenarios = new PresentationAddScenarios();
\ No newline at end of file
+// homePageScenarios = new HomePageScenarios();
```
Comment (ezequielc) [inline test/e2e/test-suite.scenarios.js]: @alex-deaconu I think we could remove the comments or would you prefer to do it after @ghislaineguerin finishes the UX? Cheers

**Comment by ezequielc** [comment]
@alex-deaconu Looks good so far. Thanks

**Comment by alex-deaconu** [comment]
@ezequielc oops, fixed the test suite. Good point re: i18n Strings, will wait until the UX is finalized. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc as discussed I've included the i18n translations for the error messages. 

Also, this PR should fix issue #135.

Thanks!

**Comment by ezequielc** [comment]
@alex-deaconu :+1:

**Pull request merged**